docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p><strong>Brennan:</strong></p><blockquote>"s/giveHint/hint/ ?"</blockquote><p><br></p><p><strong>Bart:</strong></p><blockquote>"Oh like rename this to something else like `var s` or `var hint`?</blockquote>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>